confignode/src/main/java/org/apache/iotdb/confignode/service/ConfigNode.java

```diff
@@ -47,15 +52,19 @@ public class ConfigNode implements ConfigNodeMBean {
 
   private final RegisterManager registerManager = new RegisterManager();
 
-  private final ConfigNodeRPCService configNodeRPCService;
-  private final ConfigNodeRPCServiceProcessor configNodeRPCServiceProcessor;
+  private static ConfigNodeRPCService configNodeRPCService;
```

Comment:
why change static?

Reply:
> why change static?

because the Class ConfigNode  have not initial.